app/config/config.neon

```diff
@@ -17,26 +17,44 @@ parameters:
       prefixLength: 2 # splitting hash files into 256 subdirectories by prefix should be enough
 
   api:
-    address: https://recodex.mff.cuni.cz:4000
+    address: "https://your.recodex.domain/api"
     name: "ReCodEx API"
     description: "ReCodEx Code Examiner - API Server"
     versionFormat: "{tag}"
 
+  async:
+    pollingInterval: 60 # seconds (you may set this to larger values if inotify wakeups are allowed)
+    retries: 3 # how many times each async job is retried when failing
+    # inotify can wake the async worker (immediately once an async opertion is issued)
+    inotify: false # set to true only if your system (and PHP) supports inotify (not available on Windows, extension required on Linux)
+    inotifyFile: %tempDir%/async-inotify-file # file used as inotify rod for triggering async worker on dispatch
```

Comment:
I think that it would be better to add magic word `recodex` somewhere to the path, so it's clear to what app this file belongs to...

Also what does rod mean in this case?

Reply:
The tempDir is not global tmp, but local temp in the application directory (same place where Nette caches are).

The file acts like a *lightning rod* -- collects the touches (lightnings) and conducts them to a safe place (select sync operation). Perhaps this metafore has gone too far this time.

Reply:
Oh, I did not realize that about tempDir.